scout/server/blueprints/institutes/templates/overview/gene_variants.html

```diff
@@ -87,6 +90,31 @@
 {% endmacro %}
 
 
+{% macro zygosity(variant) %}
+  {%- for sample in variant.samples -%}
+    {%- for ind in variant.case.individuals -%}
+      {%- if sample.sample_id == ind.individual_id -%}
+        {% set allele_depths = ['ref depth', sample.allele_depths[0]]|join(":") + ' - ' + ['alt depth', sample.allele_depths[1]]|join(":") %}
```

Comment:
If this is exactly the same code as in causatives and verified, why not having a common macro, shared by the 3?

Reply:
Unfortunately not quite, due to the case object. Buuut I'm sure we could generalise it.